undeclared UINT_MAX

Following the default steps to compile. I've encountered this error while running `make`
```bash
  CC       src/openfortivpn-config.o
  CC       src/openfortivpn-hdlc.o
  CC       src/openfortivpn-http.o
  CC       src/openfortivpn-io.o
  CC       src/openfortivpn-ipv4.o
  CC       src/openfortivpn-log.o
  CC       src/openfortivpn-tunnel.o
  CC       src/openfortivpn-main.o
src/main.c: In function 'main':
src/main.c:310:44: error: 'UINT_MAX' undeclared (first use in this function); did you mean 'WINT_MAX'?
     if ((persistent < 0) || (persistent >= UINT_MAX)) {
                                            ^~~~~~~~
                                            WINT_MAX
src/main.c:310:44: note: each undeclared identifier is reported only once for each function it appears in
make: *** [Makefile:590: src/openfortivpn-main.o] Error 1
```

I've noticed that `#include <limits.h>` wasn't in the main.c
After adding it everything went well again

--- a/src/main.c
+++ b/src/main.c
@@ -25,6 +25,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <limits.h>
 
 #define usage \
 "Usage: openfortivpn [<host>:<port>] [-u <user>] [-p <pass>]\n" \

